phpunit.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
phpunit has been moved to here and made similar to other apps, so that the phpunit CI works like other apps